### PR TITLE
task-1: modified content assets

### PR DESCRIPTION
**Pull request by Vladzio26**

### CODE REVIEW

File: storefront-reference-architecture-master/metadata/task1.xml [hunk 1]
```diff
@@ -0,0 +1,13204 @@
...
+&lt;p&gt;&lt;b&gt;Analogico:&lt;/b&gt; Un televisore analogico non può trasmettere i programmi in alta definizione. È in grado di trasmettere solo programmi creati con una definizione standard, come quelli della televisione normale, della televisione via cavo o dei canali satellitari, inclusi il digitale terrestre, DirectTV o Dish Network negli Stati Uniti.       &lt;br /&gt;&#13;
+&lt;br /&gt;&#13;
+&lt;b&gt;Digitale:&lt;/b&gt; Per "televisione digitale" si intende la programmazione a definizione standard (SDTV), la programmazione a definizione avanzata (EDTV) o la programmazione ad alta definizione (HDTV).        &lt;br /&gt;&#13;
+&lt;br /&gt;&#13;
+&lt;b&gt;SDTV:&lt;/b&gt; La televisione a definizione standard è una televisione analogica con un sintonizzatore ATSC integrato (vedi sotto) che permette di ricevere la programmazione digitale. Questo tipo di televisore potrà trasmettere la programmazione ad alta definizione ma le immagini non appariranno così nitide come appaiono su un vero televisore HD.        &lt;br /&gt;&#13;
+&lt;br /&gt;&#13;
+&lt;b&gt;EDTV:&lt;/b&gt; Si tratta della televisione a definizione avanzata, un televi
```
Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task1.xml:6172]: @oleksandra-davydchuk 
Added here

Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task1.xml:6221]: @oleksandra-davydchuk 
Added here

Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task1.xml:6294]: @oleksandra-davydchuk
Added here

Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task1.xml:6370]: @oleksandra-davydchuk
Added here

**Comment by Vladzio26** [comment]
Hello @oleksandra-davydchuk, 
You can find result here at the very bottom of the page
https://zzrp-011.sandbox.us01.dx.commercecloud.salesforce.com/on/demandware.store/Sites-RefArch-Site

**Comment by oleksandra-davydchuk** [comment]
checked

**Pull request merged**